doc/installation/mac_install.rst

```diff
@@ -1,95 +1,120 @@
-              </path/to/NEST/src>
+         codesign -f -s "gdb-cert" `which orted`
+         codesign -f -s "gdb-cert" `which orterun`
 
- .. code-block:: sh
+      Instead of the ``which`` command you can also give the full path to the binary inside your conda
+      environment.
+      
+      .. note::
+      
+         You will need to sign the binaries every time you update the OpenMPI package in your environment.
 
-        make -j4         # -j4 builds in parallel using 4 processes
-        make install
-        make installcheck
 
-To compile NEST with MPI support, add ``-Dwith-mpi=ON`` as ``cmake`` option.
+Building NEST
+-------------
 
-Troubleshooting
-.................
+1. Download or clone the NEST sources from `<https://github.com/nest/nest-simulator>`__.
 
-If compiling NEST as described above fails with an error message like
+#. Create a build directory outside the NEST sources and move into it.
 
- .. code-block:: sh
+#. Configure NEST by running
 
-        In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/wait.h:110,
-                         from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/stdlib.h:66,
-                         from /usr/local/Cellar/gcc/9.2.0/include/c++/9.2.0/cstdlib:75,
-                         from /usr/local/Cellar/gcc/9.2.0/include/c++/9.2.0/bits/stl_algo.h:59,
-                         from /usr/local/Cellar/gcc/9.2.0/include/c++/9.2.0/algorithm:62,
-                         from /Users/plesser/NEST/code/src/sli/dictutils.h:27,
-                         from /Users/plesser/NEST/code/src/sli/dictutils.cc:23:
-        /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/resource.h:443:34: error: expected initializer before '__OSX_AVAILABLE_STARTING'
-          443 | int     getiopolicy_np(int, int) __OSX_AVAILABLE_STARTING(__MAC_10_5, __IPHONE_2_0);
-              |                                  ^~~~~~~~~~~~~~~~~~~~~~~~
+   .. code-block:: sh
 
- you most likely have installed a version of XCode prepared for the next version of macOS. You can attempt to fix this by running
+      cmake -DCMAKE_INSTALL_PREFIX:PATH=<nest_install_dir> </path/to/NEST/src>
 
-  .. code-block:: sh
+   If you have libraries required by NEST such as GSL installed with Homebrew and Conda, this
+   can lead to library conflicts (error messages like ``Initializing libomp.dylib, but found
+   libomp.dylib already initialized.``). To ensure that libraries are found first in your conda
+   environment, invoke ``cmake`` like this
 
-          sudo xcode-select -s /Library/Developer/CommandLineTools/
+   .. code-block:: sh
 
-If this does not help, you can reset to the default XCode path using
+      CMAKE_PREFIX_PATH=<conda environment path> cmake -DCMAKE_INSTALL_PREFIX:PATH=<nest_install_dir> </path/to/NEST/src>
```

Comment:
See above. Also please try to make `<conda environment path>` consistent

Reply:
Done.